src/script/calling/CallingRepository.ts

```diff
@@ -1268,6 +1267,8 @@ export class CallingRepository {
       return;
     }
 
+    this.callClosedCallback(conversationId);
```

Comment:
Oh this is called even when the user leaves on purpose. That's good refactoring then 👌